docs/home/howto/SECURITY.md

```diff
@@ -367,3 +367,257 @@ cp /etc/kubernetes/admin.conf $HOME/
+  vault_script_autoconfiguration: true # enable automatic configuration of Hashicorp Vault
+  tls_disable: false # enable TLS support, should be used always in production
+  ...
+  app_secret_path: devwebapp # application specific path where application secrets will be mounted
+  revoke_root_token: false # not implemented yet (more about in section Root token revocation)
+  secret_mount_path: secret # start of the path that where secrets will be mounted
+  vault_token_cleanup: true # should configuration script clean token
+  vault_install_dir: /opt/vault # directory where vault will be installed
+  vault_log_level: info # logging level that will be set for Vault service
+  certificate_name: fullchain.pem
+  private_key_name: privkey.pem
+  vault_tls_valid_days: 365
+  override_existing_vault_users: false # should user from vault_users ovverride existing user and generate new password
+  vault_users: # users that will be created with vault
+    - name: admin # name of the user that will be created in Vault
+      policy: admin # name of the policy that will be assigned to user (descrption bellow)
+    - name: provisioner
+      policy: provisioner
+```
+
+More information about configuration of Vault in Epiphany and some guidance how to start working with Vault with Epiphany you can find below.
+
+To get more familiarity with Vault usage you can reffer to [official getting started](https://learn.hashicorp.com/vault) guide.
+
+### Creation of user using Epiphany in Vault
+
+To create user by Epiphany please provide list of users with name of policy that should be assigned to user. You can
+use predefined policy delivered by Epiphany, default Vault policies or your own policy. Remember that if you have
+written your own policy it must exist before user creation.
+
+Password for user will be generated automatically and can be found in directory /opt/vault in files matching
+tokens-*.csv pattern. If user password will be generated or changed you will see corresponding line in csv file with
+username, policy and password. If password won't be updated you will see `ALREADY_EXISTS` in password place.
+
+### Predefined Vault policies
+
+Vault policies are used to define Role-Based Access Control that can be assigned to clients, applications and other
+components that are using Vault. You can find more information about policies [here](https://www.hashicorp.com/resources/policies-vault/).
+
+Epiphany besides two already included in vault policies (root and default) provides two additional predefined policies:
+
+- admin - policy granting administration privileges, have sudo permission on Vault system endpoints
+- provisioner - policy granting permissions to create user secrets, adding secrets, enable authentication methods, but
+  without access to Vault system endpoints
+
+### Manual unsealing of the Vault
+
+When Hashicorp Vault starts it starts in sealed mode. This mean that Vault data is encrypted and Vault needs to generate key
```

Comment:
`data is encrypted and Vault needs to generate key` ... isn't it bit incorrect? I think it should be: `data is encrypted and operator needs to provide unsealing key` or something similar.